src/skmultiflow/core/stream.py

```diff
@@ -0,0 +1,28 @@
+from streamz import Stream
+
+@Stream.register_api()
+class predict(Stream):
+    def __init__(self, upstream, model, **kwargs):
+        self.model = model
+        super().__init__(upstream, **kwargs)
+
+    def update(self, X, who=None):
+        return self._emit(self.model.predict(X))
+
+@Stream.register_api()
+class partial_fit(Stream):
+    def __init__(self, upstream, model, **kwargs):
+        self.model = model
+        super().__init__(upstream, **kwargs)
+
+    def update(self, x, who=None):
+        X, y = X
+        self.model.partial_fit(X, y)
+        return self._emit(self.model)
+
+@Stream.register_api()
+class from_sklearn_pipeline(Stream):
+    """
+    take a pipeline and build a stream
+    """
+    pass  # TODO
```

Comment:
This would function like a generator? Strictly speaking, this looks like a Source, but since I don't suppose a user would do any processing to it before passing to a learning model (it was already processed by the pipeline), not sure there would be much use in exposing this.

Reply:
This something to drop.
My actual idea is to provide a helper to get users from the sklearn comunity
That could be a function that constructs a streamz.Stream from a sklearn.Pipeline

If you already have a pipeline, you would call something like
```python
stream = Stream.from_sklearn_pipeline(pipeline)
```

I agree there is a confusion with the term "Pipeline". A stream can actually be seen as a pipeline. That kind of helper could be used to improve interactions from both worlds (static pipelines from sklearn, streams from streamz)

This is only an idea for now. I'll remove this and save it for later